src/otx/api/entities/label_schema.py

```diff
@@ -41,9 +41,9 @@ def natural_sort_label_id(target: Union[ID, LabelEntity, ScoredLabel]) -> List:
 
     if isinstance(target, (LabelEntity, ScoredLabel)):
         target = target.id_
-    if isinstance(target, int):
-        return [target]
-    return [int(t) if t.isdigit() else t for t in re.split(r"(\d+)", target)]
+    if isinstance(target, str) and target.isdecimal():
+        return ["", int(target)]  # "" is added for the case where id of some lables is None
```

Comment:
Could you help me understand? If `target.id_` is `None`, the output will be `[None]`. Then, will it be compared against other cases like `["", 1234]` or `["a1234b1234"]`?

Reply:
For the first question, if `target.id_` is None, then it's converted to `""`. Then it can make error when comparing `""` and `integer` because they have different type. 
For the second question, yes. If we assume that we get two labels each of which has `1234` and `a1234b1234`. Then keys for sorting are `["", 1234]` and `["a1234b1234"]`. Because `["", 1234]` <  `["a1234b1234"]`, `1234` prior to `a1234b1234`. But this case is not the case we consider as right case. I suppose that two cases can be given to us. First case, all labels have decimal format string or None (OTX) and second case all labels have string format or None (Geti).